src/library-authoring/author-library/LibraryAuthoringPage.jsx

```diff
@@ -103,25 +104,34 @@ export const BlockPreviewBase = ({
         </Button>
       </Navbar.Collapse>
     </Navbar>
-    <Modal
-      open={showDeleteModal}
-      title={intl.formatMessage(messages['library.detail.block.delete.modal.title'])}
+    <ModalDialog
+      isOpen={showDeleteModal}
       onClose={() => setShowDeleteModal(false)}
-      body={(
+    >
+
+      <ModalDialog.Header>
+        <ModalDialog.Title>
+          {intl.formatMessage(messages['library.detail.block.delete.modal.title'])}
+        </ModalDialog.Title>
+      </ModalDialog.Header>
+      <ModalDialog.Body>
         <div>
           <p>
```

Comment:
do we need the `<div>` and the `<p>` here? my assumption would be that the `<div>` was needed before because the body wasn't a component, but now that we have `<ModalDialog.Body>` it might be redundant.

Reply:
Resolved